contributing.md

```diff
@@ -0,0 +1,146 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Filing an issue
+
+1. Look for a similar issue that may already describe the proposed changes.
+
+1. If none exists, create a new issue and describe the proposed changes.
```

Comment:
I can't think of a better phrase right now but the issue shouldn't describe the proposed changes. It should describe the bug, problem, desired feature, etc.

Reply:
@peterbe That's what I meant by change, not implementation.

Reply:
@peterbe I removed the word 'change' from the issue process entirely.